docs/database/server/server.md

```
@@ -446,3 +446,59 @@ title string
 channel_id string INDEX FK >- Channel.id
 reply_count number
 update_at number INDEX
+
+
+BoardView
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)